src/workerd/io/worker.c++

```diff
@@ -1194,6 +1194,35 @@ Worker::Isolate::Isolate(kj::Own<Api> apiParam,
                   v8::Local<v8::Value> argument)> callback) -> v8::Maybe<void> {
         try {
           auto& js = jsg::Lock::from(isolate);
+
+          {
+            // TODO(soon): Hopefully this logging is temporary. We want to get an idea
+            // of where the cross request promises are being resolved just in general.
+            // To do so we need to try to capture a stack. We do so by creating an error
+            // object and logging it.
+            v8::HandleScope handleScope(isolate);
+            auto err =
+                v8::Exception::Error(js.str("Cross Request Promise Resolve"_kj)).As<v8::Object>();
+            jsg::check(err->Set(js.v8Context(), js.str("name"_kj), js.str("Warning"_kj)));
+            auto stack = jsg::check(err->Get(js.v8Context(), js.str("stack"_kj)));
+            // Let's only log if there's actually a useful stack in the log.
+            auto msg = kj::str("NOSENTRY ", stack);
+            // This check is meant to prevent spamming the logs with messages that do
+            // not actually provide any useful information.
+            if (msg != "NOSENTRY Warning: Cross Request Promise Resolve") {
```

Comment:
```suggestion
            // This check is meant to prevent spamming the logs with messages that do
            // not actually provide any useful information.
            if (stack != "Warning: Cross Request Promise Resolve") {
```